crates/node-core/src/args/debug.rs

```diff
@@ -28,6 +28,38 @@ pub struct DebugArgs {
     #[arg(long = "debug.max-block", help_heading = "Debug")]
     pub max_block: Option<u64>,
 
+    /// Runs a fake consensus client that advances the chain using recent block hashes
+    /// on Etherscan. If specified, requires an `ETHERSCAN_API_KEY` environment
+    #[arg(
+        long = "debug.etherscan",
+        help_heading = "Debug",
+        conflicts_with = "tip",
+        conflicts_with = "rpc_consensus_http"
+    )]
+    pub etherscan: bool,
```

Comment:
this should take an `Option<Option<String>>` if Some(Some(api_url)) use that, if Some(None) use the chain's default

Reply:
done!